lib/utils/mbaas-token-check.js

```diff
@@ -1,23 +1,31 @@
-var _ = require('underscore');
-var common = require("../common");
-var fhreq = require("./request");
+const _ = require('underscore');
+const common = require("../common");
+const fhreq = require("./request");
 
-module.exports = {
-  check: function(request, cb) {
-    var url = '/api/v2/mbaases/' + request.target;
-    common.doGetApiCall(fhreq.getFeedHenryUrl(), url, i18n._('Error checking MBaaS type'), function(err, response) {
-      if (err) {
-        return cb(err);
-      }
-      // Ensure that a token is provided if creating an openshift3 environment
-      if (response.type === 'openshift3' && typeof request.token === 'undefined') {
-        return cb(i18n._('OpenShift targets require a token to be provided'));
-      }
-      if (response.type === 'feedhenry' && request.token) {
-        return cb(i18n._('FeedHenry targets do not accept tokens'));
-      }
+function checkMbaasType(request, cb) {
+  const url = '/api/v2/mbaases/' + request.target;
+  common.doGetApiCall(fhreq.getFeedHenryUrl(), url, i18n._('Error checking MBaaS type'), function checkIfTokenRequired(err, response) {
+    if (err) {
+      return cb(err);
+    }
+
+    if (response.type === 'openshift3') {
+      request.targetType = 'openshift3';
+    }
+
+    // Ensure that a token is provided if creating an openshift3 environment
+    if (response.type === 'openshift3' && typeof request.token === 'undefined') {
+      return cb(i18n._('OpenShift targets require a token to be provided'));
+    }
 
-      return cb(null, request);
-    });
-  }
+    if (response.type === 'feedhenry' && request.token) {
+      return cb(i18n._('FeedHenry targets do not accept tokens'));
+    }
+
+    return cb(null, request);
+  });
 };
+
+module.exports = {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)